src/__test__/unit/builtInHandler.spec.ts

```diff
@@ -93,4 +124,41 @@ describe('Tests for default handler', () => {
             result: 'The JS implementation of Peer does not support identify',
         });
     });
+
+    it('sign should work', async () => {
+        // arrange
+        const ctx = await context;
+        const req: CallServiceData = {
+            serviceId: 'security',
+            fnName: 'sign',
+            args: [testData],
+            tetraplets: [
+                [
+                    {
+                        function_name: 'get_trust_bytes',
+                        json_path: '',
+                        peer_pk: '',
+                        service_id: 'TrustGraph',
+                    },
+                ],
+            ],
+            particleContext: {
+                particleId: 'some',
+                initPeerId: ctx.peerId,
+                timestamp: 595951200,
+                ttl: 595961200,
+                signature: 'sig',
+            },
+        };
+
+        // act
+        const fn = builtInServices(ctx)[req.serviceId][req.fnName];
+        const res = await fn(req);
+
+        // assert
+        expect(res).toMatchObject({
+            retCode: 0,
+            result: testDataSig,
+        });
```

Comment:
maybe also check signature via `security verify`?

Reply:
There are two tests for verification:

```
  ${'security'} | ${'verify'}          | ${[testData, testDataSig]}      | ${0}    | ${true}}
  ${'security'} | ${'verify'}          | ${[testData, testDataWrongSig]} | ${0}    | ${false}}
```

Reply:
Yeah, but they all work with static data. Let's have at least a single test that does actual verification of the signature.

P.S. I'm not sure what's the purpose of tests with static signatures (except the wrongSig, of course)

Reply:
Signing and verification are pure functions. There are tests for different sets of input\output parameters which cover positive and negative use cases for both of these functions. I don't understand your point about "actual verification". The test for `verify` actually tests if it is verifying the data correctly or not. In theory I can calculate the signature on the piece of paper and pass it to the verification functions. It doesn't care where the sig came from